plugins/modules/identity/ipa/ipa_user.py

```diff
@@ -293,8 +295,9 @@ def get_ssh_key_fingerprint(ssh_key, hash_algo='sha256'):
     if len(parts) < 3:
         return "%s (%s)" % (key_fp, key_type)
     else:
-        user_host = parts[2]
-        return "%s %s (%s)" % (key_fp, user_host, key_type)
+        comment_elements = parts[2:]
```

Comment:
I think a better fix would be to change line 283 to `parts = ssh_key.strip().split(None, 2)`. This will make sure that `len(parts) <= 3`.

Reply:
That is a better solution - I have never used split that way, but after looking it up it will do the job better.  This patch assumes all whitespace was spaces, which could also be wrong.  I will update the PR later today.

Reply:
The limit parameter of `split()` is really useful in cases like this :)

Please note that you cannot use the kwargs version of the second argument with Python 2, so both `None` and `2` need to be specified so it works for both Python 2 and Python 3.

Reply:
Update pushed to better implementation.  Thanks for the comment!